qkernel/src/kernel_def.rs

```diff
@@ -235,7 +237,7 @@ pub fn HyperCall64(type_: u16, para1: u64, para2: u64, para3: u64, para4: u64) {
 #[inline(always)]
 pub fn HyperCall64(type_: u16, para1: u64, para2: u64, para3: u64, para4: u64) {
     let vcpuid = GetVcpuId();
-    let share_para_page  = MemoryDef::HYPERCALL_PARA_PAGE_OFFSET as *mut ShareParaPage;
+    let share_para_page  = MemoryDef::hcall_page_gpa() as *mut ShareParaPage;
```

Comment:
I forgot to ask in the previous PR. But are you sure the HyperCall interface is synchronized? Otherwise this hypercall param shared buffer should be on a per-cpu (or thread) basis.